tn_msdp_api_v1.yml

```diff
@@ -0,0 +1,1736 @@
+openapi: "3.0.0"
+
+info:
+    version: 1.0.0
+    title: Trip Ninja API Documentation
+    description: |
+        <h2>Introduction</h2>
+        Trip Ninja uses a REST API with requests and responses in JSON format. For development and testring, the pre-production servers
+        are to be used. Once ready for live use, access to the production servers will be provided. All search endpoints have both production
+        and pre-production endpoints - these correspond to the GDS’s production and pre-production services, and can be used as desired.<br  />
+        <h2>Trip Ninja API URLs</h2>
+        
+        Pre-Production: https://preprodapi.tripninja.io
```

Comment:
One question:
The url will looks like this?
`https://preprodapi.tripninja.io/preprod/` or `https://preprodapi.tripninja.io/prod/`?

Suggestion:
URL: `https://preprod.api.tripninja.io/` 
`{endpoint}`: I think we should be able to get `endpoint` variable on the backend side.

Reply:
in the docs the endpoints will look like this, so i think the current setup is alright 
![image](https://user-images.githubusercontent.com/99681496/205775599-ed7cab94-c4d0-414f-916a-85a79f83b39d.png)

Reply:
I don't mean the document itself.
Let's assume that we are API users.
To send the request to our API, the url will looks like this - `https://preprodapi.tripninja.io/preprod/`.
Here we have double `preprod` on the url and I think it should be updated.
What do you think?

Reply:
done! it only shows prod now! @flutistar